codon_program4.py:4:47-52: error: name 'bytes' is not defined